mcs_benchmark_data/pipelines/mcscript/trian_mcscript_submission_pipeline.py

```diff
@@ -10,19 +11,26 @@
 )
 
 
-class TrianMCScriptSubmissionPipeline(_Pipeline):
+class TrianMCScriptSubmissionPipeline(_SubmissionPipeline):
     BENCHMARK_ID = "mcscript"
     SUBMISSION_ID = "trian"
 
-    def __init__(self, data_dir_path: Path = DATA_DIR_PATH, **kwds):
-        _Pipeline.__init__(
+    def __init__(
+        self,
+        *,
+        data_dir_path: Path = DATA_DIR_PATH,
+        dataset_type: str = _SubmissionPipeline.DATASET_TYPE_DEFAULT,

```

Comment:
Having a _SubmissionPipeline was a good idea, because you can have this dataset_type default in that.